.Rbuildignore

```diff
@@ -3,3 +3,5 @@ Makefile
 .travis.yml
 appveyor.yml
 paper.md
+^.*\.Rproj$
+^\.Rproj\.user$
```

Comment:
You don't need to add these to the PR. It should just be the things needed to fix the issue you are worried about.